cmake/config.cmake

```diff
@@ -285,6 +285,9 @@ set(USE_HEXAGON_SDK /path/to/sdk)
 # Whether to build the hexagon launcher
 set(USE_HEXAGON_LAUNCHER OFF)
 
+# Whether to build the minimal support android rpc server for hexagon
+set(USE_MINIMAL_HEXAGON_TVM_RPC OFF)
```

Comment:
Maybe make this more explicit? `USE_HEXAGON_PROXY_RPC`?

Reply:
also you could add it to tvm/CMakeList.txt as a `tvm_option` with some description.

Reply:
Great point, I've added these changes in [8d0e7d7](https://github.com/apache/tvm/pull/9526/commits/8d0e7d7faa30bd1c1a748561e796fa704b7fb7ed).